src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsDetailsMapping.java

```diff
@@ -156,21 +157,28 @@ public static boolean shouldBeHearingsInWelshFlag() {
 
     public static List<HearingLocations> getHearingLocations(CaseManagementLocation caseManagementLocation) {
         HearingLocations hearingLocations = new HearingLocations();
-        hearingLocations.setLocationId(caseManagementLocation.getBaseLocation());
-        hearingLocations.setLocationType("court");
+        List<HearingLocations> location;
+        List<String> multipleLocationList = new ArrayList<>(List.of("Manchester", "Chester", "Plymouth"));
 
         List<HearingLocations> hearingLocationsList = new ArrayList<>();
-        hearingLocationsList.add(hearingLocations);
+
+        if (multipleLocationList.contains(caseManagementLocation.getRegion())) {
+            location = GetVenueMultipleEpims.getMultipleLocationDetails(caseManagementLocation);
+            hearingLocationsList.addAll(location);
+        } else {
+            hearingLocations.setLocationId(caseManagementLocation.getBaseLocation());
```

Comment:
Above should mean you don't need to do this if check, it should also greatly simplify GetVenueMultipleEpims